junit-jupiter-params/src/test/kotlin/ParameterizedTestNameFormatterIntegrationTests.kt

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.MethodOrderer
+import org.junit.jupiter.api.TestInfo
+import org.junit.jupiter.api.TestMethodOrder
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.MethodSource
+
+@TestMethodOrder(MethodOrderer.MethodName::class)
+class ParameterizedTestNameFormatterIntegrationTests {
+    @BeforeEach
+    fun setUp(info: TestInfo) {
+        System.out.printf(
+            "Class: %s, method: %s%nDisplay Name: %s%n%n",
+            info.testClass.orElseThrow(),
+            info.testMethod.orElseThrow(),
+            info.displayName
+        )
+    }
```

Comment:
We do not print to SYSOUT in tests for debugging purposes.

So, please remove this method.